### PR TITLE
Add documentation for passing a Regexp as a Process Name

### PR DESCRIPTION
**Pull request by directionless**

### CODE REVIEW

**Comment by directionless** [comment]
I found myself needing to use the source code to figure this out. So, trivial doc patch.

(Yes, I've signed a CLA)

File: docs/resources/processes.md.erb [hunk 1]
```diff
@@ -16,7 +16,7 @@ A `processes` resource block declares the name of the process to be tested, and
 
 where
 
-* `processes('process_name')` must specify the name of a process that is running on the system
+* `processes('process_name')` specifies the of a process to check. If this is a string, it will be converted to a Regexp. For more specificity, pass a Regexp directly.
```
Comment (adamleff) [inline docs/resources/processes.md.erb:19]: I think this should be `specifies the name of a process to check` - looks like "name" was dropped accidentally.

**Review by adamleff — CHANGES_REQUESTED** [review]
Thanks for your submission, @directionless - just one small tweak for accuracy is needed.

**Comment by chris-rock** [comment]
@directionless Thank you for your addition! Once you signed off the commit (https://github.com/chef/inspec/blob/master/CONTRIBUTING.md#dco-sign-off-methods) and changed the text according to @adamleff feedback, we are able to merge this.

**Comment by adamleff** [comment]
@chris-rock given the age of this PR and that this is a doc-only fix, I'm going to adopt this PR on behalf of the user and get it merged under our obvious-fix / documentation-only fix exception.

**Comment by chris-rock** [comment]
@adamleff Alright, makes sense.

**Comment by adamleff** [comment]
Superceded by #1761

**Comment by directionless** [comment]
Sorry for dropping this! I'm not very good about github notifications.

I think probably would have fallen under the Obvious Fix Policy, but I'm glad @adamleff adopted it. 

Yay

**Pull request closed without merging**